p4rt_app/utils/table_utility.cc

```diff
@@ -15,28 +15,52 @@
 
 #include "absl/status/status.h"
 #include "gutil/status.h"
+#include "absl/strings/substitute.h"
 #include "p4_pdpi/utils/annotation_parser.h"
 #include "swss/schema.h"
 
 namespace p4rt_app {
 namespace table {
 
+// This is what orch agent understands as fixed tables
+std::vector<std::string> FixedTables {
+  APP_P4RT_ROUTER_INTERFACE_TABLE_NAME,
+  APP_P4RT_NEIGHBOR_TABLE_NAME,
+  APP_P4RT_NEXTHOP_TABLE_NAME,
+  APP_P4RT_WCMP_GROUP_TABLE_NAME,
+  APP_P4RT_IPV4_TABLE_NAME,
+  APP_P4RT_IPV6_TABLE_NAME,
+  APP_P4RT_MIRROR_SESSION_TABLE_NAME,
+  APP_P4RT_L3_ADMIT_TABLE_NAME,
+  APP_P4RT_TUNNEL_TABLE_NAME
+};
```

Comment:
This becomes a maintainable list that needs to be updated for any new FIXED table addition, can we instead try to follow a convention like have a EXT pre/post fix that we can use to identify the table type ?

Reply:
I am thinking if there is a way to use supported schema to extract these tables. So, that way we don't have to have this list. What do you think?

Reply:
That sounds doable as well.

Reply:
ok, I'll work on that

Reply:
I have addressed this one. One caveate, there are few build time tests that expect certain table-names and table-types without making use of Schema, and so for now added those 3 tables in initialization list so those tests can pass

Reply:
A global vector that is populated dynamically is not looking clean here, this has to be re-factored.

Reply:
can you please elaborate what do you have in mind?

Reply:
Lets define 2 static lists here, existing set of fixed tables and ext tables and use that to identify the table type.